index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/8c5a2b9118.js" crossorigin="anonymous"></script>
+    <title>Portfolio Setup</title>
+</head>
+<body>
+    <header>
+        <p>John Doe</p>
+        <button class="header-btn"><i class="fas fa-bars"></i></button>
+    </header>
+
+    <div class="container">
+        <ul class="headline">
+            <li class="primary-text">
+                <h1>Hey There. <br> I’m Alex</h1>
+            </li>
+            <li class="supporting-text">
+                <h2>I’am a Software Developer</h2>
+            </li>
+            <li class="description">
+                <h4>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</h4>
+            </li>
+            <ul class="social-media">
+                <li><i class="fa fa-brands fa-github"></i></li>
+                <li><i class="fa fa-brands fa-linkedin"></i></li>
+                <li><i class="fa fa-brands fa-twitter"></i></li>
+                <li><i class="fa fa-brands fa-twitter"></i></li>
+                <li><i class="fa fa-brands fa-medium"></i></li>
+            </ul>
+        </ul>
```

Comment:
1- Consider adding a `a tag` in each of `li tags`. Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href attribute`.

2- There exists a difference in the size of icons as compared to the respective [Figma template.](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27&t=dqLqteKgCvgwyC45-0) Try making it closer to them.
![image](https://user-images.githubusercontent.com/94223367/211999962-e54afbf2-b27e-4bae-a7f3-74e4c0621389.png)
![image](https://user-images.githubusercontent.com/94223367/212000017-fbf701ef-bae9-4840-9e79-94d3b3f8e5e8.png)